packages/query-core/src/queryClient.ts

```diff
@@ -526,6 +527,10 @@ export class QueryClient {
       defaultedOptions.networkMode = 'offlineFirst'
     }
 
+    if (defaultedOptions.queryFn === skipToken) {
+      defaultedOptions.enabled = false
+    }
```

Comment:
Nice, I agree

Reply:
i think the implementation is fine as-is. If you pass `enabled: true`, it will bypass `skipToken` and then error out when the queryFn is executed. I think it doesn't make sense to set both, and this behaviour is okay.